### PR TITLE
results: Add sdm660 results extracted from Xiaomi Mi A2

### PR DESCRIPTION
**Pull request by Tashar02**
These are the benchmark results extracted from Xiaomi Mi A2 that has a Snapdragon 660 (SDM660) running on linux-4.19.217 using freqbench.

### CODE REVIEW

File: results/README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ These are reference freqbench results that have been collected from various devi
 | sdm630     | SDM630    | Snapdragon 630  | Sony Xperia 8           | Unknown  |
 | sdm636     | SDM636    | Snapdragon 636  | Xiaomi Redmi Note 5/Pro | Unknown  |
 | sdm660     | SDM660    | Snapdragon 660  | Xiaomi Redmi Note 7     | Unknown  |
+| sdm660     | SDM660    | Snapdragon 660  | Xiaomi Mi A2            | Unknown  |
```
Comment (kdrag0n) [inline results/README.md:24]: Additional devices with the same SoC aren't currently tracked separately in the index, so there's no need to add a duplicate here.

File: results/sdm660/third/cmdline.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+rcupdate.rcu_normal_after_boot=1 quiet noirqdebug nodebugmon kpti=off nosoftlockup=0 mce=ignore_ce cgroup_disable=pressure androidboot.hardware=qcom user_debug=31 msm_rtb.filter=0x37 ehci-hcd.park=3 lpm_levels.sleep_disabled=1 service_locator.enable=1 androidboot.configfs=true androidboot.usbcontroller=a800000.dwc3 loop.max_part=7 printk.devkmsg=on usbcore.autosuspend=7 kpti=off buildvariant=userdebug veritykeyid=id:7e4333f9bba00adfe0ede979e28ed1920492b40f rcu_nocbs=0-7 isolcpus=1-7 nohz_full=1-7 loglevel=0 root=/dev/dm-0 dm="system none ro,0 1 android-verity /dev/mmcblk0p65" androidboot.verifiedbootstate=orange androidboot.keymaster=1  androidboot.veritymode=enforcing androidboot.bootdevice=c0c4000.sdhci androidboot.serialno=REDACTED androidboot.cpuid=REDACTED androidboot.baseband=msm mdss_mdp.panel=1:dsi:0:qcom,mdss_dsi_nt36672_tianma_fhd_video:config0:1:none:cfg:single_dsi androidboot.slot_suffix=_a skip_initramfs rootwait ro init=/init androidboot.dtb_idx=-1347440721 androidboot.fpsensor=fpc androidboot.secureboot=1 androidboot.hwc=Global androidboot.hwversion=2.21.0 androidboot.hwlevel=MP
```
Comment (kdrag0n) [inline results/sdm660/third/cmdline.txt:1]: Please rename `third` to the device codename, `jasmine`, as it's not the same device as `main`.

**Review by kdrag0n — CHANGES_REQUESTED** [review]
Thanks! Just a few minor changes to make because this is an additional/secondary device that's different from the first one contributed for this SoC.

File: results/README.md (hunk 1 above)
Comment (Tashar02) [reply results/README.md:24]: Ok :)

File: results/sdm660/third/cmdline.txt (hunk 2 above)
Comment (Tashar02) [reply results/sdm660/third/cmdline.txt:1]: Hmm, ok👍

Comment (Tashar02) [reply results/sdm660/third/cmdline.txt:1]: Full codename is "jasmine_sprout", should I add this or just "jasmine"?

Comment (kdrag0n) [reply results/sdm660/third/cmdline.txt:1]: Just "jasmine" is fine.

**Comment by Tashar02** [comment]
@kdrag0n you can check now :)

**Review by kdrag0n — APPROVED** [review]

**Comment by kdrag0n** [comment]
Looks good, thanks!

**Comment by Tashar02** [comment]
You're welcome!

**Pull request merged**